### PR TITLE
first commit for Adding Eggs Benny's Feb Recipe

### PR DESCRIPTION
**Pull request by hamza-m-masood**
Changed index.md, asparagus-with-cranberries-and-pine-nuts.md, and eggs-benny.md.

### CODE REVIEW

File: app/index.md [hunk 1]
```diff
@@ -2,8 +2,6 @@
 
 Welcome to _Flavor_, the only place on the planet where your taste buds won't be disappointed!
 
-
-
```
Comment (jams100) [inline app/index.md:6]: WRONG Do it again sir

Comment (hamza-m-masood) [reply app/index.md:6]: okay sir, sorry sir.

File: app/index.md [hunk 2]
```diff
@@ -2,8 +2,6 @@
 
 Welcome to _Flavor_, the only place on the planet where your taste buds won't be disappointed!
```
Comment (jams100) [inline app/index.md:3]: I'll pull yours, you pull mine. OK?

Comment (hamza-m-masood) [reply app/index.md:3]: ok, lovely sir.

**Comment by jams100** [comment]
Ready to commit

**Pull request merged**